modules/cli-installing-cli.adoc

```diff
@@ -73,9 +73,9 @@ ifdef::openshift-origin[]
 . Download `oc.tar.gz`.
 endif::[]
 ifndef::openshift-origin[]
-. Navigate to the link:https://cloud.redhat.com/openshift/install[Infrastructure Provider] page on the {cloud-redhat-com} site.
-. Select your infrastructure provider, and, if applicable, your installation type.
-. In the *Command line interface* section, select *Linux* from the drop-down menu and click *Download command-line tools*.
+. Navigate to the link:https://access.redhat.com/downloads/content/290[{product-title} downloads page] on the Red Hat Customer Portal.
+. Select the appropriate version in the *Version* drop-down menu.
```

Comment:
when I open the 'Red Hat Customer Portal' I saw : Active Subscription Required
 and can't find the 'version' drop-down menu. Is this expected ?

Reply:
@zhouying7780 No that's not expected. Are you logged in to the portal (with an account that has subscriptions)?

I assume that you would have had to log in to the old location (in cloud.redhat.com/console.redhat.com) with an account with subscriptions to be able to download the CLI there too.

Reply:
@zhouying7780 Let me know if you still see the issue when you are logged in. thanks!